dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts

```diff
@@ -58,18 +59,28 @@ export class FalcoOrgSettingsPageComponent implements OnInit {
   }
 
   removeRule(ruleId: number, idx: number) {
-    this.falcoService.deleteRule(ruleId)
-      .pipe(take(1))
-      .subscribe({
-        next: () => {
-          this.alert.success('Rule deleted');
-          // Splice will remove the element and move other elements up.
-          // It is slow for large arrays, but these arrays will likely not be large enough for that to be a concern
-          this.rules.splice(idx, 1);
-        },
-        error: (err) => {
-          this.alert.danger(err?.error?.metadata || 'Something went wrong.');
-        }
-      });
+    const openAddCluster = this.dialog.open(AlertDialogComponent, {
+      width: '400px',
+      closeOnNavigation: true,
+      disableClose: false,
+      data: {},
+    });
+    openAddCluster.afterClosed().subscribe(shouldDelete => {
```

Comment:
The after closed subscription should have .pipe(1)

Reply:
Recheck :)

Reply:
Nice work